cinema/serializers.py

```diff
@@ -85,9 +100,8 @@ class Meta:
 class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
         data = super(TicketSerializer, self).validate(attrs=attrs)
-        Ticket.validate_ticket(
-            attrs["row"], attrs["seat"], attrs["movie_session"]
-        )
+        Ticket.validate_ticket(attrs["row"], attrs["seat"],
+                               attrs["movie_session"])
```

Comment:
The `validate` method in `TicketSerializer` calls `Ticket.validate_ticket` with three arguments, but the `validate_ticket` method in the `Ticket` model requires four arguments, including `error_to_raise`. Ensure that the correct number of arguments is passed to `validate_ticket`.